smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -116,6 +119,15 @@ function makeReaderNameForDisplaying(readerName) {
   return readerName;
 }
 
+function updateAddDeviceButtonText(readersCount){
+  if(readersCount == 0){
+    addDeviceElement.dataset.i18n = 'addDevice';
+  }
+  else{
+    addDeviceElement.dataset.i18n = 'addAnotherDevice';
+  }
+  GSC.I18n.adjustElementsTranslation();
```

Comment:
As discussed offline, it'd be great if we only applied this to the |addDeviceElement|, rather than to the whole page. (The real-world performance gain would be negligible, but it'd still be nice to avoid unnecessary operations.)

For this, we could have a new GSC.I18n.adjustElementTranslation(element) function that calls transformElement().

Reply:
I was thinking we name it GSC.I18n.adjustSingleElementTranslation(element) to avoid confusion with GSC.I18n.adjustElementsTranslation(), since the difference would be only one letter?

Reply:
From that perspective, I'd prefer we rather rename the old function to be "adjustAllElementsTranslation", and have the new one named just "adjustElementTranslation".

Reply:
Ok sure, is this a gut feeling, or am I missing something regarding naming conventions?

Reply:
There's no hard rule. I was just thinking along these lines: "do something for one thing" is a more basic primitive that probably deserves a shorter and a simpler name, and "do something for a bunch of things" is its derivative and can have a longer name to be clearly distinguishable. Not insisting, and happy to know your opinion too.

P.S. https://martinfowler.com/bliki/TwoHardThings.html :-)

Reply:
Ok yeah that makes sense